### PR TITLE
better streamline plural up --cloud

### PR DESCRIPTION
**Pull request by zreigz**
## Summary
 - replace the cluster name field with querying all the console instances for a user, using a select to choose the one they want to use, and use that name for the cluster name

 - no longer ask for the az spread (just chose one as a default and ignore)

 - make the bucket prefix automatically plrl-cloud-${name}.

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: cmd/command/up/up.go [hunk 1]
```diff
@@ -119,3 +119,31 @@ func (p *Plural) handleUp(c *cli.Context) error {
 	utils.Highlight("Feel free to use terraform as you normally would, and leverage the gitops setup we've generated in the apps/ subfolder\n")
 	return nil
 }
+
+func getCluster(cd *cd.Plural) (id string, name string, err error) {
+	if cd == nil {
+		return "", "", fmt.Errorf("please provide a plural client")
```
Comment (michaeljguarino) [inline cmd/command/up/up.go:125]: probably should change this error message to something like:

```
You're cli is not logged into Plural, try running `plural login` to generate local credentials.
```

File: cmd/command/cd/cd_clusters.go [hunk 2]
```diff
@@ -118,19 +118,12 @@ func (p *Plural) cdClusterCommands() []cli.Command {
 }
 
 func (p *Plural) handleListClusters(_ *cli.Context) error {
```
Comment (michaeljguarino) [inline cmd/command/cd/cd_clusters.go:120]: This actually does work, but we can also use the `consoleInstances` query, which just selects Plural cloud clusters.

File: pkg/provider/aws.go [hunk 3]
```diff
@@ -159,17 +159,20 @@ func getEC2Client(ctx context.Context, region string) (*ec2.Client, error) {
 }
 
 // TODO: during Plural init we should ask the user to choose which AZs they want to use (first 3, random, manual, look at how CAPA does that). There should be a minimum limit of 3.
-func getAvailabilityZones(ctx context.Context, region string) ([]string, error) {
+func getAvailabilityZones(ctx context.Context, region string, cloudFlag bool) ([]string, error) {
```
Comment (michaeljguarino) [inline pkg/provider/aws.go:162]: could we just have this default to `first3` entirely?  It's not piped into `plural up` at all so is actually irrelevant at the moment.  Was only for CAPI if i remember

**Review by michaeljguarino — COMMENTED** [review]
only a few small nits

File: cmd/command/up/up.go (hunk 1 above)
Comment (zreigz) [reply cmd/command/up/up.go]: done

File: cmd/command/cd/cd_clusters.go (hunk 2 above)
Comment (zreigz) [reply cmd/command/cd/cd_clusters.go:120]: I searched the whole `schema.graphql` and couldn't find this query

File: pkg/provider/aws.go (hunk 3 above)
Comment (zreigz) [reply pkg/provider/aws.go]: done

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**